### DISCUSSION TITLE
apache-airflow-providers-google: google-ads-12.0.0

### DISCUSSION THREAD

**Discussion by irosario1999** [apache/airflow#17110] [Q&A] [answered]
Hey team, I was looking to use the google ads hook but it seems like the google ads package is a bit out of date with the hook only taking` "v5", "v4", "v3", "v2"`  https://developers.google.com/google-ads/api/docs/release-notes and all of those being deprecated. Is there any chance the provider can be upgraded to include this? here is the release note of https://github.com/googleads/google-ads-python/releases/tag/12.0.0  google's 12.0.0 release which also deprecated v5

**Comment by potiuk** [comment, marked as answer]
Feel free to make a PR to upgrade it ! Happy to review it @rv-irosario
